resources/lang/en/common/notifications/screen_candidates.php

```diff
@@ -0,0 +1,16 @@
+<?php
+return [
+    /*
+    |--------------------------------------------------------------------------
+    | Screen Candidates Email Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are used during screen applicants emails for
+    | the manager.
+    |*
+    |*
+ */
+    'subject' => 'Your :position :classification posting just closed - look at your applicants now',
+    'manager_portal_link' => 'https://talent.test/en/manager',
```

Comment:
These should be relative urls, or prefixed with the app url config value:
`/en/manager` or  `config('app.url') . '/en/manager'`

Reply:
I think you want `route('manager.home')` or `route('manager.jobs.index')`.

Reply:
Oh wait, these functions might not work in a lang file. I think it makes more sense to take the url out of the lang file, and put it in the ScreenCandidatesPrompt.

Reply:
> These should be relative urls, or prefixed with the app url config value:
> `/en/manager` or `config('app.url') . '/en/manager'`

Whoops, missed that! Is there a way to get a localized route (eg. `getLocalizedRoute('manager.home', 'fr')`)? Since, we need the english and french route in the email.

Reply:
Looks like
```php
LaravelLocalization::getURLFromRouteNameTranslated('en', 'manager.home');
```
might do it?

[Docs here](https://github.com/mcamara/laravel-localization#get-url-for-an-specific-translation-key)

Reply:
@Khristophor ftw 🎉. Thanks.